### ISSUE TITLE
There is bug with using version 0.11, with some icons displaying as "?"

### ISSUE THREAD

**Issue by BaiLei27** [devblackops/Terminal-Icons#127]
## Current Behavior

There is an issue with using version 0.11, with some icons displaying as "?"
if using version 0.10, this issue will not occur
## Steps to Reproduce (for bugs)

![302116179-c7e32daa-dc0c-48ed-9d2f-8c40d8593f2b](https://github.com/devblackops/Terminal-Icons/assets/65945846/e23ea678-7d25-4bab-9c25-98e2d5cb2233)

The left side of the screenshot is Terminal-Icons version 0.11.0, and the right side is version 0.10.0
The screenshot above only shows that some icons have problems, and there are more icons with problems.
## Your Environment

* Module version used:
  0.11.0
* Operating System and PowerShell version:
  Window 11 22H2
  PowerShell 7.4.1

**Comment by verzada** [comment]
I had the same problem with 0.11.0, after following @Baiyi27 report, downgrading to 0.10 worked for me as well.

My OS is
![image](https://github.com/devblackops/Terminal-Icons/assets/10551898/4984354e-65b3-4742-aa36-d586bbdaf648)
Powershell 7.4.1

**Comment by webbertakken** [comment]
For me it was the other way around. Was having some questionmarks in v0.10.0. 
When I upgraded to v0.11.0 using `Install-Module Terminal-Icons -F`, things started working.

I'm using the latest version of `FiraCode Nerd Font Mono`.

Having the reverse of @verzada makes me think that this issue might be caused by a mismatch between Nerd Font and Terminal-Icons versions. Perhaps some icon allocations switched around in the fonts and Terminal Icons adapted accordingly?

**Comment by Tiberriver256** [comment]
Yep, if 0.10.0 looks good for you and 0.11.0 doesn't. You have Nerd Fonts v2 installed.

Uninstall all your v2 fonts and then install v3 fonts to upgrade.

**Comment by rsavard-tt** [comment]
> Uninstall all your v2 fonts and then install v3 fonts to upgrade.

Can you elaborate?  The NerdFont version seems to vary with each font and is all over the place.  ( https://www.nerdfonts.com/font-downloads )

I am using the latest version of my preferred font and it doesn't display correctly with 0.11.0.

Thanks!
